### ISSUE TITLE
ACL file search skips every other level

### ISSUE THREAD

**Issue by timbl** [linkeddata/ldnode#62]
debug trace, filtered:
```
ldnode:ACL Looking for policies in /Library/WebServer/Documents/TimBL/Automation/Library/Mashup/jquery.min.js.acl +0ms
ldnode:ACL Looking for policies in /Library/WebServer/Documents/TimBL/Automation/Library/Mashup/.acl +0ms
ldnode:ACL Looking for policies in /Library/WebServer/Documents/TimBL/Automation/.acl +0ms
ldnode:ACL Looking for policies in /Library/WebServer/Documents/.acl +0ms
ldnode:ACL Found policies in /Library/WebServer/Documents/.acl +0ms
```

It never found the one in /Library/WebServer/Documents/TimBL/Automation/Library/.acl

with
commit 30dc1b525de1dc89ef2d9edd740c894b5e141d10
Author: nicola <notsecurity@gmail.com>
Date:   Mon Jul 13 00:45:33 2015 +0200

It looks like dirname() being invoked twice ...

**Comment by nicola** [comment]
Maybe @martinmr should have a look at this, I haven't gone much through the acl, yet

**Comment by martinmr** [comment]
I looked at the code in gold (https://github.com/linkeddata/gold/blob/master/acl.go#L154) and it seems to be adapted correctly in ldnode. I don't know why dirname is being called twice. Perhaps @deiu  can shed light on the issue?

**Comment by deiu** [comment]
Is this issue still relevant? Has it been fixed? (tests seem to pass)

**Comment by martinmr** [comment]
Not sure. Tim found the issue. I still don't understand why gold calls
pathfile.Dir twice.

**Comment by timbl** [comment]
The only way to be sure is to check the logs I think to make sure that really does go up the chain from bottom to top completely not missing anything out.

**Comment by timbl** [comment]
who wrote that line to call  dirname(dirname())?

**Comment by martinmr** [comment]
Fixed in #93 . Refer to the PR for an explanation of the issue.
